### ISSUE TITLE
the python_script folder is not part of the release

### ISSUE THREAD

**Issue by npbhavya** [hahnlab/CAFE#47]
I am running cafe on my data following the tutorial and noticed that the python_scripts are not part of the release I have installed. I did copy the python scripts from the cafe image on Jetstream to my computer to run the remaining steps. 

Were these script left out from the release on purpose, it would be great if they could be made available here as part of the cafe installation.

**Comment by fkmendes** [comment]
https://hahnlab.github.io/CAFE/manual.html > Tutorial Files (
https://iu.app.box.com/v/cafetutorial-files)


2018-07-13 2:15 GMT+12:00 Bhavya Papudeshi <notifications@github.com>:

**Comment by benfulton** [comment]
Yes, the Python scripts are part of the tutorial, not part of the application. Maybe in a future release we could add them in.

**Comment by npbhavya** [comment]
Awesome, thank you for the links. Yeah adding them to a new release would
be great,

Thank you,
Bhavya

**Comment by zhangwenting0816** [comment]
it seems the Tutorial files cannot be linked and download?

**Comment by benfulton** [comment]
It works fine for me. Are you unable to download the files?

**Comment by wuxiaopei0509** [comment]
the kink is not valid now ,where can I found the Tutorial files
